Source/Moya+Internal.swift

```diff
@@ -174,17 +174,17 @@ private extension MoyaProvider {
     }
 
     func sendUploadFile(_ target: Target, request: URLRequest, queue: DispatchQueue?, file: URL, progress: ProgressBlock? = nil, completion: @escaping Completion) -> CancellableToken {
-        let alamoRequest = manager.upload(file, with: request)
+        let alamoRequest = target.validate ? manager.upload(file, with: request).validate() : manager.upload(file, with: request)
```

Comment:
This also applies to the `sendDownloadRequest` and `sendRequest` functions, of course.